base/services/shsvcs/shsvcs.spec

```diff
@@ -1,16 +1,16 @@
-1 stdcall ThemeWatchForStart()
```

Comment:
Why would we not want fixed ordinals for this?

Reply:
@tkreuzer did remove ordinals for all functions of shsvcs, see this commit from where I based that new one: https://github.com/reactos/reactos/commit/82ca85ddc089d84958b1aabbdb17a83f868f2eed

Reply:
Providing ordinals is only useful for nameless exports, i.e. imported by ordinal. For everything else it doesn't make sense, since these ordinals can change between different architectures / versions and I don't think we want to write new spec files for each and every combination.

Reply:
Named ordinals, like the first 6 of shsvcs?

Reply:
Please let me know when you reach a consensus.

Reply:
Verified: first 6 are nameless exports, which should probably have ordinals + "-noname"

Reply:
@HBelusca Can you fix the first 6 to make then -noname and with ordinals?

Reply:
I will do, thanks!